Cargo.toml

```diff
@@ -69,6 +70,12 @@ bitvec = { version = "1", default-features = false, features = ["alloc"] }
 cairo-lang-starknet = { version = "2.1.0-rc2", default-features = false }
 cairo-lang-casm = { version = "2.1.0-rc2", default-features = false }
 
+cairo-lang-compiler = { version = "2.1.0-rc2", default-features = false } 
```

Comment:
here you pin the cairo compiler crates to 2.1.0-rc2 but then you compile the contracts with cairo 2.2.0, this can lead to errors

Reply:
This is actually cargo working in our favour, as depsite the version being pinned to "2.1.0-rc2", the "2.2.0" version is the one downloaded. But ill change it to avoid confusions